### ISSUE TITLE
High Priority: File upload not working for some files

### ISSUE THREAD

**Issue by mblack-dariah** [recogito/recogito-client#342]
Issues with uploading files through the ‘Add document’ > ‘Import document’ > ‘File upload’ function
It seems that files larger than 1 MB cannot be uploaded, possibly due to their size, as no explanation was provided for the upload failure. This was tested with PDFs from the ADS data sample (17 out of 42 files were uploaded).
The same seems to apply to JPEG images: a window appears displaying the file, but the process stalls and nothing happens.
<img width="578" alt="Screenshot 2024-12-18 at 15 23 50" src="https://github.com/user-attachments/assets/ae18d3fe-0d4c-45f9-9354-5cb7fd39293f" />

**Comment by mblack-dariah** [comment]
Could this also be temporarily solved through the back-end?

**Comment by rsimon** [comment]
Agreed - this does sound very much like a size limit on your internal proxy. I don't think the Supabase storage backend would enforce a size limit. (Even if you use the cloud product on the free plan, their limit is 50MB. But the above issue is with a self-hosted instance.)

**Comment by rsimon** [comment]
An HTTP 401 error means "Not Authorized". Can you provide more information about your setup? I'm guessing you are not running a dedicated IIIF server outside of Recogito, but instead using Cantaloupe + Supabase storage? (@jsmueller7 I believe this would be identical to your setup, right?) How did you configure this in your .env file? Did you confirm your Cantaloupe server is correctly set up?

**Comment by dpancic** [comment]
We are using supabase-community helm chart https://github.com/supabase-community/supabase-kubernetes/tree/main/charts/supabase that does not include Cantaloupe iiif server.
After adding Cantaloupe IIIF server and reconfiguring Kong that it can work with it, some images are working fine and some still returning 401. @carlonim, could you please try to remove 0992.jpg and add it again?

We will try to replace our current setup with the setup based on the Helmchart created by @jsmueller7.

**Comment by dpancic** [comment]
After detailed testing, we identified that the issue was caused by restoring from a backup. Specifically, restoring the storage volume is not functioning correctly due to the bug described in this GitHub issue [https://github.com/supabase/storage/issues/201](url).

The TypeScript script suggested in the issue related to the extended attribute problem did not resolve the issue. Additionally, we encountered problems when restoring the database from a dump. It appears that the database restore only works if the /var/lib/postgres directory is restored from the backup. Using pgsql or pg_restore results in errors, and users are unable to log in after the restore.

Could you please provide us with instructions on how Recogito Studio should be backed up and restored?
